rf/imaging.py

```diff
@@ -104,29 +112,36 @@ def _plot(ax, t, d, i):
     for i, tr in enumerate(stream):
         times = tr.times(reftime=tr.stats.onset)
         xlim = (min(xlim[0], times[0]), max(xlim[1], times[-1]))
-        _plot(ax1, times, tr.data / max_ * scale, i + 1)
+        if show_traces: 
+            if scale > 0:  # scale to all-trace max
+                _plot(ax1, times, tr.data / max_ * scale, i + 1)
+            elif scale < 0:  # scale trace by trace
```

Comment:
Can you please use a new kwarg for this functionality, a negative scale value is a quick fix, but not intuitive to others.